packages/makeswift-next-plugin/index.js

```diff
@@ -70,6 +70,7 @@ module.exports =
                 },
               ],
               destination: '/api/makeswift/preview',
+              locale: false,
```

Comment:
`locale: false` disables "transparent" locale handling and instructs Next to include the locale prefix in the path: https://nextjs.org/docs/pages/api-reference/config/next-config-js/rewrites#rewrites-with-i18n-support